scripts/src/osd/sdl_cfg.lua

```diff
@@ -130,6 +130,10 @@ if _OPTIONS["targetos"]=="windows" then
 			"WIN32_LEAN_AND_MEAN",
 			"NOMINMAX",
 		}
+		buildoptions
+		{
+			"-Wno-undef",
+		}
```

Comment:
What’s this necessary for?  I can build with MinGW SDL 2.0.14 on Windows without needing this flag.  What’s different about the bundled SDL 2.0.14 that causes it to need this?  Also, it doesn’t follow the formatting, and having it here will break Visual Studio builds.

Reply:
SDL_config_windows.h is causing issues with `#if _MSC_VER >= 1911` on line 195. I will fix the visual studio issue.

Reply:
Fixed.